zap/src/main/java/org/zaproxy/zap/extension/ascan/CustomScanDialog.java

```diff
@@ -902,6 +922,13 @@ public String validateFields() {
             return Constant.messages.getString("ascan.custom.notSafe.error");
         }
 
+        if (this.filterPanel != null) {
+            String fail = this.filterPanel.validateFields();
```

Comment:
`errorMessage`

Reply:
as in below custompanel block fail is used so made it consistent.

Reply:
Didn't notice that one, mind changing that too? It's more clear than "fail" (IMO).

Reply:
sure.